.github/workflows/create-release.yml

```diff
@@ -0,0 +1,50 @@
+name: create-release
+
+on: workflow_dispatch
+
+jobs:
+  setup-variables:
+    uses: ZTL-ARTCC/Current_Website/.github/workflows/setup-variables.yml@main
+
+  tag:
+    name: tag and release main
+    runs-on: "ubuntu-latest"
+    needs: setup-variables
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: check for existing release
+        run: |
+          TAG="0.2"
+          if git show-ref --tags --verify --quiet "refs/tags/${{ needs.setup-variables.outputs.tag-name }}"; then
+            echo "Tag ${{ needs.setup-variables.outputs.tag-name }} already exists. Only one release per day allowed"
```

Comment:
This seems like an arbitrary limitation. Why is this here? The ability to override merge rules and tag an emergency release is something we should not lock ourselves out of

Reply:
If anything... if 20240724 already exists, do a 20240724-patch1 or something

Reply:
Good point... I'll see what I can do

Reply:
This is resolved and squashed into 04223c3. I additionally added a step to output the tag in a summary when it runs